content/en/blog/_posts/2022-04-28-cni-version-related-breaking-change.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: blog
+title: "Breaking changes related to CNI version in popular container runtimes"
+date: 2022-04-28T12:00:00+0000
+slug: cni-version-related-breaking-change-k8s-1.24
+canonicalUrl: https://kubernetes.dev/blog/2022/04/28/cni-version-related-breaking-change-k8s-1.24/
+---
+
+**Authors & Contributors:** [Mike Brown](https://github.com/mikebrow), [Rolfe Dlugy-Hegwer](https://github.com/rolfedh),[Nate W.](https://github.com/nate-double-u), and others.
```

Comment:
```suggestion
**Authors & Contributors:** [Mike Brown](https://github.com/mikebrow), [Rolfe Dlugy-Hegwer](https://github.com/rolfedh),[Nate Waddington](https://github.com/nate-double-u), and others.
```

Reply:
Based on a previous request from @sftim and your previous comments, I've simplified this list. Let me know if you want our name to appear nevertheless.